classes/author.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'item'
+
+class Author
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def add_item(item)
+    item.set_author(self)
+  end
```

Comment:
Please pull the changes from the other branch. I have already done a code review on this.